docs/src/modules/utils/CodeCopy.tsx

```diff
@@ -133,6 +133,21 @@ function InitCodeCopy() {
   return null;
 }
 
+function hasNativeSelection(element: HTMLTextAreaElement) {
+  if (window.getSelection()?.toString() !== '') {
```

Comment:
Actually, this is wrong
```suggestion
  if (window.getSelection() && window.getSelection().toString() !== '') {
```

Reply:
Nice catch! Although I think this is much better:
```suggestion
  if (window.getSelection()?.toString()) {
```
Since empty string is falsy.